modules/simple_forms_api/app/services/simple_forms_api/s3/submission_archive_builder.rb

```diff
@@ -129,8 +116,12 @@ def form_data_hash
         @form_data_hash ||= JSON.parse(submission.form_data)
       end
 
+      # Name the form PDFs and/or individual submission folders
+      # uniquely, using a field that also appears in the manifest.
+      # The recommended format is Form-number-vagov-submission ID
       def submission_pdf_filename
-        @submission_pdf_filename ||= "form_#{form_data_hash['form_number']}.pdf"
+        form_number = form_data_hash['form_number']
+        @submission_pdf_filename ||= "form_#{form_number}_vagov_#{benefits_intake_uuid}.pdf"
```

Comment:
I'm using underscores here given that both the form_number and benefits_intake_uuid will contain dashes.